lib/tsc.js

```diff
@@ -1,14 +1,14 @@
 /*! *****************************************************************************
-Copyright (c) Microsoft Corporation. All rights reserved. 

```

Comment:
Do not change this file manually. this is a generated file from the build, updated by running Jake LKG.